tensorflow_datasets/core/dataset_builder.py

```diff
@@ -1238,4 +1337,5 @@ def _build_pcollection(pipeline):
       return beam_writer.write_from_pcollection(pcoll_examples)
 
     # Add the PCollection to the pipeline
-    _ = pipeline | split_name >> _build_pcollection()   # pylint: disable=no-value-for-parameter
+    # pylint: disable=no-value-for-parameter
```

Comment:
Why move the comment ? This makes pylint disable globally the check, rather than only for this line.

Reply:
Undone